boxes.cc

```diff
@@ -173,13 +173,13 @@ void BoxesTest::Boxes(const std::string &_physicsEngine, double _dt,
 
       auto model = models[model_no];
       link = model->GetLink();
-      // linear velocity 
+      // linear velocity in world frame
       ignition::math::Vector3d v = link->WorldCoGLinearVel();
-      // angular velocity
-      ignition::math::Vector3d a = link->WorldAngularVel();
+      // angular velocity in body frame
+      ignition::math::Vector3d a = link->RelativeAngularVel();
```

Comment:
the [boxes_msg.proto](https://github.com/yaswanth1701/benchmark/blob/post-processing/proto/boxes_msg.proto#L70-L71) definition says that the twist is all in the world frame. I noticed that the post-processing calculation is slightly easier if we record the body-fixed angular velocity, but I think it's clearer to log everything in the world frame and make the appropriate calculations in the post-processing script